### PR TITLE
[NIFI-13165] hide resize handle is status history when insufficient h…

### PR DESCRIPTION
**Pull request by scottyaslan**
…istory
Screenshot 2024-05-07 at 2 35 03 PM
Screenshot 2024-05-07 at 2 35 18 PM

### CODE REVIEW

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/status-history.component.html [hunk 1]
```diff
@@ -14,7 +14,13 @@
...
 ~  limitations under the License.
 -->
-<div resizable (resized)="resized()" [minHeight]="720" [minWidth]="887" class="flex flex-col status-history-container">
+<div
+    resizable
+    [resizeDisabled]="instances.length === 0 || fieldDescriptors.length === 0"
+    (resized)="resized()"
+    [minHeight]="720"
+    [minWidth]="887"
+    class="flex flex-col status-history-container">
```
Comment (rfellows) [inline nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/status-history.component.html:23]: Not specific to your changes, but while you are in here... does it make sense to update the logic when deciding to show/hide the refresh/last loaded timestamp in the footer?
Currently it only bases its decision on the number of instances and ignores the number of fieldDescriptors so we end up with an odd dialog like so:
Screenshot 2024-05-08 at 10 35 06 AM

technically they can keep refreshing so it isn't terrible as it is. just not ideal. i suppose if they hit refresh and there happens to be data at that point, the dialog would just grown and look like the other.

what are your thoughts here, @scottyaslan?

Comment (scottyaslan) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/status-history.component.html:23]: Sure, I can hide it in this case.

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/status-history.component.html [hunk 2]
```diff
@@ -170,7 +170,7 @@ <h2 mat-dialog-title>Status History</h2>
             <mat-dialog-actions align="start">
                 <div class="flex w-full">
                     <div class="flex-1">
-                        @if (instances.length > 0) {
+                        @if (instances.length === 0 || fieldDescriptors.length === 0) {
```
Comment (rfellows) [inline nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/status-history.component.html:173]: this is inverted. should be 
```suggestion
                        @if (instances.length > 0 && fieldDescriptors.length > 0) {
```

**Review by rfellows — CHANGES_REQUESTED** [review]

**Review by rfellows — APPROVED** [review]
LGTM 👍 
Will merge once CI completes

**Pull request merged**